### PR TITLE
`BUG FIX` Update User Guide bugs

### PR DESCRIPTION
**Pull request by nareus**
# Bugs Fixed
- [PE-D][Tester A] description for adding multiple tags is not included #245
- [PE-D][Tester E] Wrong command format in User Guide #236
- [PE-D][Tester D] Incorrect specification of filter usage in UG #232

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -610,18 +611,20 @@ RC4HDB saves your data after every command. There is no need to save manually.
 ---
 
 ### File Command Format:
-* All file commands have a `FOLDER_NAME` field, which signifies the name of the **sub** data folder or the [CSV](#glossary-of-terms) file to be targeted depending on the command that is being used.
-* Do not include any file types in the `FOLDER_NAME`.
-* The following symbols are not to be used in `FOLDER_NAME`.
+* All file commands have a `FOLDER_NAME` or `FILE_NAME` field depending on the command that is being used. 
+* `FOLDER_NAME` signifies the name of the **sub** data folder that is being used.
+* `FILE_NAME` refers to the [CSV](#glossary-of-terms) file to be targeted.
+* Do not include any file types in the `FOLDER_NAME` or `FILE_NAME`.
```
Comment (nealetham) [inline docs/UserGuide.md:617]: Should we use the term file extension? Is that too technical for the user?

Comment (nealetham) [reply docs/UserGuide.md:617]: On second thought, it might be too technical. One way we can dumb it down is to state that the file type is anything beyond the dot i.e. .json

**Review by nealetham — CHANGES_REQUESTED** [review]
Just a minor change required.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -614,7 +614,7 @@ RC4HDB saves your data after every command. There is no need to save manually.
 * All file commands have a `FOLDER_NAME` or `FILE_NAME` field depending on the command that is being used. 
 * `FOLDER_NAME` signifies the name of the **sub** data folder that is being used.
 * `FILE_NAME` refers to the [CSV](#glossary-of-terms) file to be targeted.
-* Do not include any file types in the `FOLDER_NAME` or `FILE_NAME`.
+* Do not include any file types in the `FOLDER_NAME` or `FILE_NAME`. A file type is the description that comes after the`.` in a file. For example, The file name of the file `residents.csv` is  just `residents`.
```
Comment (jq1836) [inline docs/UserGuide.md:617]: Minor issue here with the "the`.`" part, there should be a whitespace.

**Review by jq1836 — APPROVED** [review]
LGTM! Just that minor typo.

File: docs/UserGuide.md (hunk 2 above)
Comment (jq1836) [reply docs/UserGuide.md:617]: Also, there is an extra whitespace towards the end.

".... `residents.csv` is  just `residents`."

**Review by nealetham — APPROVED** [review]
Looks good!
